internal/contentcache/contentcache.go

```diff
@@ -36,6 +40,34 @@ type ContentCache struct {
 	mtimeClock     timeutil.Clock
 }
 
+// Populates the cache with existing persisted files when gcsfuse starts
+func (c *ContentCache) PopulateCache() {
+	if c.tempDir == "" {
+		c.tempDir = "/tmp"
+	}
+	files, err := ioutil.ReadDir(c.tempDir)
+	if err != nil {
+		log.Fatal(err)
+	}
+	for _, file := range files {
+		// validate not a directory and matches gcsfuse pattern
+		if !file.IsDir() && matchPattern(file.Name()) {
+			// TODO ezl: load the files from disk to the in memory map
+			fmt.Printf(file.Name())
```

Comment:
This is for debug purpose, which should not be merged into the master. We don't want this message to show up in a production environment distracting the user.